### PR TITLE
V2.11.1

### PR DESCRIPTION
**Pull request by hbaqai**
Updating build.gradle files with latest OpenTok and Android versions

### CODE REVIEW

File: Simple-Multiparty/build.gradle [hunk 1]
```diff
@@ -15,9 +15,8 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
-        maven {
-            url  "http://tokbox.bintray.com/maven"
-        }
+        maven { url 'https://maven.google.com' }
```
Comment (marinaserranomontes) [inline Simple-Multiparty/build.gradle]: @hbaqai  why do we need the Google Maven repository ?

Comment (hbaqai) [inline Simple-Multiparty/build.gradle]: I upgraded to the latest appcompat support library: `com.android.support:appcompat-v7:25.4.0`

Google says to add their maven repo to import this: https://developer.android.com/topic/libraries/support-library/setup.html

If I remove this, then I get the compile error: 

`Error:(28, 13) Failed to resolve: com.android.support:appcompat-v7:25.4.0`

Comment (marinaserranomontes) [inline Simple-Multiparty/build.gradle]: @hbaqai  I think if you download the sdk tool version with the SDK manager, probably, you will not need to add the repo in the gradle file

Comment (marinaserranomontes) [inline Simple-Multiparty/build.gradle]: Agree. "The support libraries are now available through Google's Maven repository. You do not need to download the support repository from the SDK Manager."

**Comment by marinaserranomontes** [comment]
LGTM.

**Pull request merged**